.jenkins/nightly.groovy

```diff
@@ -60,12 +61,13 @@ pipeline {
                           rm -rf spack && \
                           git clone https://github.com/spack/spack.git && \
                           . ./spack/share/spack/setup-env.sh && \
-                          spack install kokkos@develop+cuda+wrapper+tests cuda_arch=80 ^cuda@12.1.0 && \
+                          spack install --only=dependencies kokkos@develop+cuda+wrapper+tests cuda_arch=80 ^cuda@12.1.0 && \
+                          spack install --only=package --cdash-upload-url="https://my.cdash.org/submit.php?project=Kokkos" --cdash-track=Nightly --cdash-site="ornl-jenkins" --cdash-build="spack-cuda" kokkos@develop+cuda+wrapper+tests cuda_arch=80 ^cuda@12.1.0 && \
```

Comment:
Please briefly explain how the split thing works

Reply:
`spack install --only=dependencies` installs all the dependencies of Kokkos (things like CMake). Then `spack install --only=package` installs Kokkos and reports the results to CDASH. If we don't split the installation process, we report to CDASH the results for all the packages. It's nice to have but it adds about 10 reports.